packages/hurl_core/src/parser/url.rs

```diff
@@ -22,7 +22,7 @@ use crate::parser::reader::Reader;
 use crate::parser::{expr, ParseResult};
 
 pub fn url(reader: &mut Reader) -> ParseResult<Template> {
-    // Can not be JSON-encoded, nor empty.
+    // Must be neither JSON-encoded nor empty.
```

Comment:
This general wording dates to the first commit in the repository.

I think this rephrasing improves things...

Reply:
Yes, we'll probably revisit the comment